digital_image_processing/test_digital_image_processing.py

```diff
@@ -19,116 +17,94 @@
 from digital_image_processing.filters import sobel_filter as sob
 from digital_image_processing.resize import resize as rs
 
-img = imread(r"digital_image_processing/image_data/lena_small.jpg")
-gray = cvtColor(img, COLOR_BGR2GRAY)
+# Sample image path for testing
+IMG_PATH = r"C:/Users/USER/Python/digital_image_processing/image_data/lena_small.jpg"
```

Comment:
Our GitHub Actions-based tests run on Ubuntu Linux so they are not going to find files on a hard-coded Windows path.  Faster but failing the tests is not going to be accepted.